app/serializers/search/postgres_comment_serializer.rb

```diff
@@ -40,9 +40,8 @@ class PostgresCommentSerializer < ApplicationSerializer
 
     # NOTE: not using the `NestedUserSerializer` to avoid hitting Redis to
     # fetch the cached value
-    attribute :user do |comment, params|
-      user = params[:users][comment.user_id]
-      user.slice(:name, :profile_image_90, :username).symbolize_keys
+    attribute :user do |comment|
+      comment.user.slice(:name, :profile_image_90, :username).symbolize_keys
```

Comment:
let's throw in a `if user` here too, as we do on line 7 and call it a day :D

Reply:
Great call, updated 💯